src/task1/idsignificantchanges/idsignificantchanges.py

```diff
@@ -0,0 +1,138 @@
+import json
+import tkinter
+import argparse
+import PIL.Image, PIL.ImageTk
+from PIL import ImageFile
+ImageFile.LOAD_TRUNCATED_IMAGES = True
+
+#written by Alex Zhuang zhual@utschools.ca 2019/02/23
+
+parser = argparse.ArgumentParser(description='You must specify paths to two jsons')
+parser.add_argument("--map", help = 'valid path to the reference map json', type = str)
+parser.add_argument("--photos", help = 'valid path to the survey photos json', type = str)
+args = parser.parse_args()
+mappath = args.map
+photopath = args.photos
+
+with open(mappath, 'r') as f:
+    inputmap = json.load(f)
+with open(photopath, 'r') as f:
+    inputsurveyphotos = json.load(f)
+
+global index, basewidth, out
+index = 0
+basewidth = 300
+
+out = {'damaged':[]}
+
+def output():
+    global out
+    with open('idsignificantchanges.json', 'w') as outfile:
+        json.dump(out, outfile)
+    top.destroy()
+
+
+top = tkinter.Tk()
+top.protocol("WM_DELETE_WINDOW", output)
+
+
+img = PIL.Image.open(inputmap['filename']);
+photo =PIL.ImageTk.PhotoImage(img)
+
+top.geometry('{}x{}'.format(photo.width(), photo.height()) )
+
+canvas = tkinter.Canvas(top, width = photo.width(), height = photo.height())
+canvas.create_image(0, 0, image=photo, anchor=tkinter.NW)
+
+canvas.pack()
+
+
+
+def clicked(event):
+    if (event.x >= 0 and event.x <= photo.width() and event.y >= 0 and event.y <= photo.height()):
+        second = tkinter.Toplevel(width = 300)
+
+        x1, y1 = (event.x - 1), (event.y - 1)
+        x2, y2 = (event.x + 1), (event.y + 1)
+
+        #Assumed that image is oriented North
+        #(relative y of point in image * distance between latitude coordinates) + most southern latitude coordinate
+        lat = (((photo.height()-event.y)/photo.height())*abs(inputmap['topleft']['lat']-inputmap['bottomright']['lat']))+inputmap['bottomright']['lat']
+        #(relative x of point in image * distance between longitude coordinates) + most western longitude coordinate
+        long = ((event.x/photo.width())*abs(inputmap['topleft']['long']-inputmap['bottomright']['long']))+inputmap['topleft']['long']
+        point = canvas.create_oval(x1, y1, x2, y2, fill="#FF0000", outline="#FF0000", width=10)
+        second.grab_set()
+        second.title("New Label")
+
+        def delete():
+            canvas.delete(point)
+            second.destroy()
+
+        msg = tkinter.Message(second, justify = 'center', width = 300, text="Label at (lat,long) \n{}, {}".format(lat, long))
+        msg.pack()
+
+        t = tkinter.Text(second, height=5, width=40)
+        t.pack()
+
+        def save():
+            global index
+            desc = t.get("1.0", 'end-1c')
+            out['damaged'].append({'lat': lat,
+                                    'long': long,
+                                    'message': desc,
+                                    'filename': inputsurveyphotos[index]})
+            print (out)

```

Comment:
nit

Reply:
uhhh what does this mean 😅

Reply:
Just another word for `I'm nit picking here and I am totally aware of that`

That is, a small change that may not be very _important_, but is technically correct. In this case, this print statement looks like it was used for debugging and forgot to be removed 👍